kafka-admin/src/main/java/org/bf2/admin/kafka/admin/ConsumerGroupOperations.java

```diff
@@ -504,21 +520,25 @@ static void validatePartitionResettable(Map<TopicPartition, List<MemberDescripti
     private static Types.ConsumerGroupMetrics calculateGroupMetrics(List<Types.Consumer> consumersList) {
         var metrics = new Types.ConsumerGroupMetrics();
 
-        var laggingConsumers = 0;
+        var laggingPartitions = 0;
         var unassignedPartitions = 0;
 
         for (Types.Consumer consumer : consumersList) {
```

Comment:
@MikeEdgar Logic counts lag only for active consumers.
I'm not sure if from point of view of user I want to see total amount of lagging consumers even disconnected or not.

Reply:
Ok, my brain wasn't working when I looked at this yesterday :-) I agree, it probably doesn't make much sense to tally that for partitions not actively being consumed.

Reply:
No worries. I always like to have discussion on those things because this metric is not obvious (and somehow as user I would probably like to have total lag of my all consumers summed up to see the system. Partitions with lag but small one like 4-10ms is actually common use case so we will be confusing people a lot.